content/blog/recipes-1-1-0/index.md

```diff
@@ -0,0 +1,325 @@
+---
+output: hugodown::hugo_document
+
+slug: recipes-1-1-0
+title: recipes 1.1.0
+date: 2024-07-01
+author: Emil Hvitfeldt
+description: >
+    recipes 1.1.0 is on CRAN! recipes now has better input checking and quality of life errors.
+
+photo:
+  url: https://unsplash.com/photos/close-up-photo-of-baked-cookies-OfdDiqx8Cz8
+  author: Food Photographer | Jennifer Pallian
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [tidymodels, recipes]
+rmd_hash: 2d6d3d7956133624
+
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with [`hugodown::tidy_show_meta()`](https://rdrr.io/pkg/hugodown/man/use_tidy_post.html))
+* [x] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] [`hugodown::use_tidy_thumbnails()`](https://rdrr.io/pkg/hugodown/man/use_tidy_post.html)
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] [`usethis::use_tidy_thanks()`](https://usethis.r-lib.org/reference/use_tidy_thanks.html)
+-->
+
+We're thrilled to announce the release of [recipes](https://recipes.tidymodels.org/) 1.1.0. censored is a parsnip extension package for survival models.
```

Comment:
Looks like this needs a re-knit! See "censored is a..."

Reply:
whooops